.github/workflows/unittest.yml

```
@@ -0,0 +1,34 @@
+name: Python package
+
+on:
+  push:
+  pull_request:
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.6','3.7', '3.8']
+
+    steps:
+      - uses: actions/checkout@v1
```

Let's upgrade this action: 
```suggestion
      - uses: actions/checkout@v3
```